lib/vmdb/productization.rb

```diff
@@ -24,7 +24,7 @@ def prepare_asset_paths
     #   The original value is located at:
     #     https://github.com/rails/rails/blob/v3.2.15/railties/lib/rails/application/configuration.rb#L48-L49
     def prepare_asset_precompilation
-      Rails.application.config.assets.precompile = [
+      Rails.application.config.assets.precompile += [
```

Comment:
Does that mean this can now just add `%(productization.css productization.js)`, and skip the proc?